src/test/scala/akka/stream/contrib/TimedSpec.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.contrib
+
+import akka.stream.scaladsl.{ Flow, Keep }
+import akka.stream.testkit.scaladsl.{ TestSink, TestSource }
+
+import scala.concurrent.duration.Duration
+
+class TimedSpecAutoFusingOn extends { val autoFusing = true } with TimedSpec
+class TimedSpecAutoFusingOff extends { val autoFusing = false } with TimedSpec
+
+trait TimedSpec extends BaseStreamSpec {
+
+  "Timed Source" should {
+
+    import akka.stream.contrib.Implicits.TimedFlowDsl
+
+    "measure time it between elements matching a predicate" in {
+
+      val measureBetweenEvery = 5
+      val printInfo = (interval: Duration) ⇒ {
+        info(s"Measured interval between $measureBetweenEvery elements was: $interval")
+      }
+
+      val flow = Flow[Int].map(identity).timedIntervalBetween(_ % measureBetweenEvery == 0, printInfo)
```

Comment:
can't you use `val testActor = TestProbe()` and send to that from `printInfo`?
it doesn't have to verify the actual timing, just that things are emitted